pkg/controller/certificates/issuing/issuing_controller_test.go

```diff
@@ -999,12 +999,9 @@ func TestIssuingController(t *testing.T) {
 			test.builder.Init()
 			defer test.builder.Stop()
 
-			// Instantiate/setup the controller
 			w := controllerWrapper{}
-			w.Register(test.builder.Context)
+			_, _, _ = w.Register(test.builder.Context)
```

Comment:
Should fail the test if the err is not nil
```suggestion
			_, _, err := w.Register(test.builder.Context)
			require.NoError(t, err)
```

Reply:
Done in 069084a17.